packages/cli/src/Generate.ts

```diff
@@ -174,7 +173,6 @@ ${bold('Examples')}
         typedSql,
         noEngine:
           Boolean(args['--no-engine']) ||
-          Boolean(args['--data-proxy']) || // legacy, keep for backwards compatibility
           Boolean(args['--accelerate']) || // legacy, keep for backwards compatibility
```

Comment:
These two should also go away (both `--data-proxy` and `--accelerate` are aliases for `--no-engine`)

Reply:
if you don't want to deal with removing the `--no-engine` flag in this PR, remove just `--accelerate`

Reply:
Ooh, i see why you removed the `--data-proxy` flag here, you probably `grep`ed the corresponding flag from the test runner across the codebase and found this one, although it's a different thing with the same name.

In that case maybe keep them all here, and remove all three in the PR where you'll be removing the `DataProxyEngine` (or any other follow up PR).